bin/lib/zopen-build

```diff
@@ -688,35 +682,23 @@ extractTarBall()
   tarballz="$1"
   dir="$2"
 
-  printInfo "Extract tarball ${tarballz} into ${dir}"
-  ext=${tarballz##*.}
-  if [ "${ext}x" = "xzx" ]; then
-    if ! xz -d "${tarballz}"; then
-      printError "Unable to use xz to decompress ${tarballz}"
-    fi
-    tarball=${tarballz%%.xz}
-  elif [ "${ext}x" = "gzx" ]; then
-    if ! gunzip "${tarballz}"; then
-      printError "Unable to use gunzip to decompress ${tarballz}"
-    fi
-    tarball=${tarballz%%.gz}
-  elif [ "${ext}x" = "bz2x" ]; then
-    if ! bunzip2 "${tarballz}"; then
-      printError "Unable to use bunzip2 to decompress ${tarballz}"
-    fi
-    tarball=${tarballz%%.bz2}
-  else
-    printError "Extension ${ext} is an unsupported compression technique. Add code"
+  if ! tar --version >/dev/null 2>/dev/null; then
+    printError "The tar open source package is required by zopen-build. The z/OS default 'tar' is insufficient"
   fi
-
-  tar -xf "${tarball}" 2>&1 >/dev/null | grep -v FSUM7171 >/dev/null
-  if [ $? -gt 1 ]; then
-    printError "Unable to untar ${tarball}"
+  tarraw=$(tar --version | head -1 | awk '{print $4;}')
+  v=${tarraw%%.*}
+  r=${tarraw##*.}
+  if [ ${v} -lt 1 ] || [ ${r} -lt 34 ]; then
+    printError "Need to be running at least tar 1.34"
   fi
-  rm -f "${tarball}"
 
-  # tar will incorrectly tag files as 1047, so just clear the tags
-  chtag -R -r "${dir}"
+  printInfo "Extract tarball ${tarballz} into ${dir}"
+
+  tar -axf "${tarballz}"  
```

Comment:
cool! I didn't know about -a

Reply:
Me either. Learned about it when perusing the new man pages.